base/Dockerfile

```diff
@@ -14,5 +14,4 @@ RUN chmod +x /usr/bin/prepare.sh
 
 RUN mkdir /opt/app
 
-ENTRYPOINT ["/usr/local/bin/dumb-init", "--"]
-CMD ["bash", "-c", "/usr/bin/prepare.sh && exec dask-scheduler"]
+ENTRYPOINT ["/usr/local/bin/dumb-init", "bash", "/usr/bin/prepare.sh"]
```

Comment:
Yes, this is definitely the solution to free the `CMD` that you mention in #4 i didn't do this initially because it would make it impossible to skip the `prepare.sh` but in general its not a big issue i think.